kolkokrzyzykV2.py

```diff
@@ -0,0 +1,84 @@
+def restart():
+    answer = input('Do you want to play again?\nType "yes" or "no".\n')
+    if answer == 'yes':
+        tictactoe()
+    elif answer == 'no':
+        exit()
+    else:
+        print('Answer not recognized.\n')
+        restart()
+
+
+def tictactoe():
+    while True:
+        cells = '         '
+        who_moves = 1
+        cells_list = [x for x in cells]
+
+        def grid():
+            if who_moves % 2 == 1:
+                player = 'X'
+            else:
+                player = 'O'
+            print(f'\nNow player {player} moves.')
+            print("---------")
+            print('| ' + cells_list[0] + ' ' + cells_list[1] + ' ' + cells_list[2] + ' |')
+            print('| ' + cells_list[3] + ' ' + cells_list[4] + ' ' + cells_list[5] + ' |')
+            print('| ' + cells_list[6] + ' ' + cells_list[7] + ' ' + cells_list[8] + ' |')
+            print("---------")
+
+        def check():
+            row1 = cells_list[0:3]
+            row2 = cells_list[3:6]
+            row3 = cells_list[6:9]
+            column1 = [cells_list[0], cells_list[3], cells_list[6]]
+            column2 = [cells_list[1], cells_list[4], cells_list[7]]
+            column3 = [cells_list[2], cells_list[5], cells_list[8]]
+            diagonal1 = [cells_list[0], cells_list[4], cells_list[8]]
+            diagonal2 = [cells_list[2], cells_list[4], cells_list[6]]
+            win_con_x = ['X', 'X', 'X']

```

Comment:
Ta zmienna pełni role stałej i nie powinna być modyfikowania, w pytonie nie istniej coś takiego jak stała ale żeby pokazać ze jest to stało to zapisuje się nazw zmiennej wielkimi literami np WIN_CON_X. Tutaj znajdziesz fajny opis [1]

[1]https://www.programiz.com/python-programming/variables-constants-literals